ethportal-api/src/types/content_key.rs

```diff
@@ -59,6 +61,10 @@ impl Serialize for HistoryContentKey {
                 let selector = "03";
                 serializer.serialize_str(&format!("0x{selector}{hex_bytes}"))
             }
+            HistoryContentKey::Unknown(value) => {
+                let hex_bytes = hex::encode(value.as_ssz_bytes());
+                serializer.serialize_str(&format!("0x{hex_bytes}"))
```

Comment:
just curious about the "correct" way to do this.  I found that it seems we can do `format!("{:x?}", value.as_ssz_bytes())` to get hex encoded values in strings...  I don't know enough yet to know whether there's a reason to use one over the other.

Reply:
One difference is that the `?` uses a `Debug` implementation. Where a `Debug` derivation may have been stripped from a `--release` build, it may now be included which could increase binary size (though this is probably not significant). 

From https://rust-lang.github.io/rfcs/2226-fmt-debug-hex.html:

> The std::fmt::Debug trait (used with {:?}) however is intended for formatting “in a programmer-facing, debugging context”. It can be derived, and doing so is recommended for most types.

Reply:
> just curious about the "correct" way to do this. I found that it seems we can do `format!("{:x?}", value.as_ssz_bytes())` to get hex encoded values in strings... I don't know enough yet to know whether there's a reason to use one over the other.

Not sure what is more idiomatic, there are also [UpperHex](https://doc.rust-lang.org/std/fmt/trait.UpperHex.html) and [LowerHex](https://doc.rust-lang.org/std/fmt/trait.LowerHex.html) traits that can be implemented on any type.